initialise_server.sql

```diff
@@ -66,6 +66,73 @@ CREATE TABLE IF NOT EXISTS properties (
 	FOREIGN KEY (property_position) REFERENCES property_values(property_position)
 );
 
+CREATE TABLE IF NOT EXISTS cards (
+    unique_id tinyint UNSIGNED NOT NULL AUTO_INCREMENT,
+	card_type ENUM('chance','chest') NOT NULL,
+	description VARCHAR(255) NOT NULL,
+	operation ENUM('move_specific', 'get_money', 'pay_bank', 'pay_opponents', 'collect_from_opponents', 'pay_per_house') NOT NULL,
+    -- operation_value will mean different things (e.g. money, position) depending on the "operation"
+    operation_value smallint,
+	PRIMARY KEY (unique_id)
+);
+
+INSERT INTO property_values
+VALUES (1, 60, 'property', 2, 50, 10, 30, 90, 160, 250),
+       (3, 60, 'property', 4, 50, 20, 60, 180, 320, 540),
+       (6, 100, 'property', 6, 50, 30, 90, 270, 400, 550),
+       (8, 100, 'property', 6, 50, 30, 90, 270, 400, 550),
+       (9, 120, 'property', 8, 50, 40, 100, 300, 450, 600),
+       (11, 140, 'property', 10, 100, 50, 150, 450, 625, 750),
+       (13, 140, 'property', 10, 100, 50, 150, 450, 625, 750),
+       (14, 160, 'property', 12, 100, 60, 180, 500, 700, 900),
+       (16, 180, 'property', 14, 100, 70, 200, 550, 750, 950),
+       (18, 180, 'property', 14, 100, 70, 200, 550, 750, 950),
+       (19, 200, 'property', 16, 100, 80, 220, 600, 800, 1000),
+       (21, 220, 'property', 18, 150, 90, 250, 700, 875, 1050),
+       (23, 220, 'property', 18, 150, 90, 250, 700, 875, 1050),
+       (24, 240, 'property', 20, 150, 100, 300, 750, 925, 1100),
+       (26, 260, 'property', 22, 150, 110, 330, 800, 975, 1150),
+       (27, 260, 'property', 22, 150, 110, 330, 800, 975, 1150),
+       (29, 280, 'property', 24, 150, 120, 360, 850, 1025, 1200),
+       (31, 300, 'property', 26, 200, 130, 390, 900, 1100, 1275),
+       (32, 300, 'property', 26, 200, 130, 390, 900, 1100, 1275),
+       (34, 320, 'property', 28, 200, 150, 450, 1000, 1200, 1400),
+       (37, 350, 'property', 35, 200, 175, 500, 1100, 1300, 1500),
+       (39, 400, 'property', 50, 200, 200, 600, 1400, 1700, 2000);
```

Comment:
I don’t think this is meant to be here, is it? (The `property_values` table is already initialised below.)